### PR TITLE
Mike

### PR DESCRIPTION
**Pull request by mvanhaver**
Is this the correct repo for the pull request?

### CODE REVIEW

File: app/src/main/java/edu/metrostate/cardealer/inventory/Vehicle.java [hunk 1]
```diff
@@ -41,6 +41,9 @@ public Vehicle(String vehicleID, String dealershipID, String vehicleType, String
 		rented = false;
 	}
 
+	public Vehicle(String toString, String s) {
```
Comment (tomatogrove) [inline app/src/main/java/edu/metrostate/cardealer/inventory/Vehicle.java:44]: could you remove this method?

File: app/src/main/java/edu/metrostate/cardealer/CarDealerApplication.java [hunk 2]
```diff
@@ -9,6 +9,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import edu.metrostate.cardealer.inventory.Vehicle;
+
 public class CarDealerApplication extends Application {
     private final List<Vehicle> vehicleList = new ArrayList<>();
```
Comment (tomatogrove) [inline app/src/main/java/edu/metrostate/cardealer/CarDealerApplication.java:15]: could you pull master? this class is outdated

File: app/src/main/java/edu/metrostate/cardealer/activity/EditSpecificVehicle.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import edu.metrostate.cardealer.CarDealerApplication;
+import edu.metrostate.cardealer.R;
+import edu.metrostate.cardealer.inventory.Vehicle;
+import edu.metrostate.cardealer.adapter.VehicleAdapter;
+
```
Comment (tomatogrove) [inline app/src/main/java/edu/metrostate/cardealer/activity/EditSpecificVehicle.java:18]: are you using both this activity and the vehicleListActivity to display a list of vehicles?

Comment (tomatogrove) [reply app/src/main/java/edu/metrostate/cardealer/activity/EditSpecificVehicle.java:18]: Also please change the name of this class to include Activity in the name

Comment (mvanhaver) [reply app/src/main/java/edu/metrostate/cardealer/activity/EditSpecificVehicle.java:18]: I tried to use the template that the professor had for the vehicle list, but I don't think that the vehicleListActivity is being used along with the EditSpecificVehicleActivity. (although I could be wrong)

**Pull request closed without merging**